app/components/works/admin_component.html.erb

```diff
@@ -7,8 +7,11 @@
       <select id="adminFunctionsSelect" class="form-select" onchange="this.value !== 'select' ? document.querySelector('#workAdminSection').src = this.value : document.querySelector('#workAdminSection').innerHTML = ''">
         <option value='select' selected>Select...</option>
         <option value='<%= edit_owners_path(work) %>'>Change owner</option>
+        <option value='<%= edit_lock_work_path(work) %>'>Lock</option>
       </select>
     </div>
   </div>
-  <turbo-frame id="workAdminSection"></turbo-frame>
-</section>
\ No newline at end of file
+  <div class="row pt-3">
```

Comment:
I don't think you want to put this here because for a non-admin, the padding still displays, but the frame may not have anything in it.

Reply:
This entire component should not render unless you are admin, and this padding is contained in the body of that component where the action specific content appears: https://github.com/sul-dlss/happy-heron/blob/main/app/components/works/admin_component.rb#L12-L14

Reply:
I see. This is great then.